fireant/dataset/data_blending.py

```diff
@@ -0,0 +1,108 @@
+from fireant import (
+    Field,
+)
+from fireant.dataset.klass import (
+    DataSet,
+)
+from fireant.queries import (
+    DimensionChoicesQueryBuilder,
+    DimensionLatestQueryBuilder,
+)
+from fireant.queries.builder.dataset_blender_query_builder import DataSetBlenderQueryBuilder
+from fireant.utils import immutable
+
+
+class DataSetBlender(object):
+    """
+    WRITEME
```

Comment:
Would be nice to have a brief description of the intended use of this class, given that it's somewhat high level